okhttp/src/main/java/okhttp3/Response.kt

```diff
@@ -65,12 +63,27 @@ class Response internal constructor(
    * * It may be the request generated in response to an HTTP redirect or authentication
    *   challenge. In this case the request URL may be different than the initial request URL.
    */
```

Comment:
Move the docs from the deprecated functions to the corresponding vals?

Reply:
For sure! Thanks for pointing it out!